paddle/fluid/framework/ir/mkldnn/onednn_replace_pass.cc

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/fluid/framework/ir/mkldnn/onednn_replace_pass.h"
+#include "paddle/fluid/framework/ir/graph_pattern_detector.h"
+#include "paddle/fluid/framework/op_version_registry.h"
+#include "paddle/fluid/platform/enforce.h"
+#include "paddle/fluid/string/pretty_log.h"
+
+#include <unordered_map>
+
+namespace paddle {
+namespace framework {
+namespace ir {
+
+using string::PrettyLogDetail;
+
+void OneDNNReplacePass::ApplyImpl(Graph *graph) const {
+  const std::unordered_map<std::string, std::vector<std::string>>
+      elementwise_io_map = {{"Inputs", {"X", "Y"}}, {"Outputs", {"Out"}}};
+  const std::unordered_map<
+      std::string, std::unordered_map<std::string, std::vector<std::string>>>
+      in_out_map = {{"elementwise_add", elementwise_io_map},
+                    {"elementwise_mul", elementwise_io_map},
+                    {"elementwise_sub", elementwise_io_map}};
+
```

Comment:
This map is needed, because the ops inputs and outpus are needed for find and create new onednn custom op.